java/data/dialog/body.mcdoc

```diff
@@ -8,6 +8,7 @@ struct DialogBody {
 
 dispatch minecraft:dialog_body[plain_message] to struct PlainMessage {
 	/// A multiline label.
+	/// Click events in the text work as expected, and trigger `after_action` like any other action.
```

Comment:
Seems weird to have a doc comment say that something works as expected; that's the default